crossbar/controller/cli.py

```diff
@@ -94,26 +95,37 @@
 _PID_FILENAME = 'node.pid'
 
 
+def get_version(name_or_module):
+    if isinstance(name_or_module, str):
+        name_or_module = importlib.import_module(name_or_module)
+
+    try:
+        return name_or_module.__version__
+    except AttributeError:
+        return ''
+
+
 def get_node_classes():
```

Comment:
I like replacing all the "get a package's version" places with a call to a single function (i.e. this one)